keyboards/boardrun/bizarre/readme.md

```diff
@@ -0,0 +1,17 @@
+# Boardrun Bizarre
+
+![Boardrun Bizarre](https://i.imgur.com/usKQUWI.jpg)
+
+The Boardrun is a family of ortholinear keyboards with columnar stagger, inspired by the Alice layout. The Classic is a 60% layout that uses Ergodox keycap sets for compatibility, whereas the Bizarre is a 65% layout that uses standard (staggered) keycap sets in an ortholinear arrangement.
+
+The Boardrun is an evolution of the Boardwalk layout. The Boardwalk proved that the Ergodox keycap set could be adapted to more keyboards that provided a more compact design and layout flexibility. While adaptation was a design goal of the Boardwalk layout, the Boardrun was designed to distill and refine the best aspects of the current state of keyboard layouts, taking into account current trends in staggered design, the Ergodox keycap layout, and the Boardwalk's most popular layouts. To that end, a columnar staggered, ortholinear layout with alpha rotation was settled on, using the "HHKB" layout from the Boardwalk. The result is two layouts that are comfortable, familiar, and have great compatibility with keysets.
+
+-   Keyboard Maintainer: QMK Community
+-   Hardware Supported: Boardrun Bizarre Ortholinear PCB
+-   Hardware Availability: mkultra.click
+
+Make example for this keyboard (after setting up your build environment):
+
+    make boardrun/bizarre:default:flash
+
```

Comment:
Can you include "clear instructions on how to reset the board into bootloader mode" as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?

Reply:
Change made as requested in latest commit.